content/scripts/document.js

```diff
@@ -16,6 +16,28 @@ const HTML_FILENAME = "index.html";
 const getHTMLPath = (folder) => path.join(folder, HTML_FILENAME);
 const getWikiHistoryPath = (folder) => path.join(folder, "wikihistory.json");
 
+function updateWikiHistory(localeContentRoot, oldSlug, newSlug = null) {
+  function read() {
+    return JSON.parse(
+      fs.readFileSync(path.join(localeContentRoot, "_wikihistory.json"))
+    );
+  }
+  function write(data) {
+    fs.writeFileSync(
+      path.join(localeContentRoot, "_wikihistory.json"),
+      JSON.stringify(data, null, 2)
+    );
+  }
+  const all = read();
```

Comment:
I'd vote for inling those functions, imo the naming doesn't give enough extra explicitness to justify the added indirection

Reply:
Yeah, it made sense and some point and then I refactored it a bunch and didn't undo the closure functions. Will update.